CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ project(vallumd C)
 
 set (VERSION_MAJOR 0)
 set (VERSION_MINOR 1)
-set (VERSION_PATCH 1)
+set (VERSION_PATCH 2)
```

Comment:
Please don't bump VERSION_PATCH. I'll decide that, and it always has to go in a commit on its own.